src/components/MessageListMessageCompact.vue

```diff
@@ -43,6 +43,11 @@
         >
             {{ ml.formatTime(message.time) }}
         </div>
+        <span
+            v-if="message.type === ('privmsg' || 'msg')"
```

Comment:
`('privmsg' || 'msg')` this will only ever match against `privmsg`

Reply:
I will address this as well as add a commit that detects whether or not the current network has the 'away-notify' cap.